league/manager/models.py

```diff
@@ -1,40 +1,42 @@
+from enum import Enum
 from django.core.exceptions import ValidationError
 from django.db import models
-from datetime import datetime
+from django.utils import timezone
 
 
 class League(models.Model):
-    name = models.CharField(max_length=50)
+    league_name = models.CharField(max_length=50)
```

Comment:
I don't think it was that necessary here, it's pretty clear that it meant the name of the league.